packages/shorebird_cli/lib/src/shorebird_env.dart

```diff
@@ -43,6 +43,16 @@ class ShorebirdEnv {
     ).readAsStringSync().trim();
   }
 
+  set flutterRevision(String revision) {
+    File(
+      p.join(shorebirdRoot.path, 'bin', 'internal', 'flutter.version'),
+    ).writeAsStringSync(revision);
+    final snapshot = File(
+      p.join(shorebirdRoot.path, 'bin', 'cache', 'shorebird.snapshot'),
+    );
+    if (snapshot.existsSync()) snapshot.deleteSync();
+  }
```

Comment:
Should we verify that `revision` isn't equal to the current contents of the `flutter.version` file before blowing the snapshot away?

Reply:
yep good catch!